docs/sources/choose-a-view/index.md

```diff
@@ -124,18 +128,34 @@ Determining which view to use frames the rest of your investigation.
 
 The **All services** view is the default view when you first open Explore Profiles.
 
-![All services](../images/explore-profiles-homescreen.png)
+![The Services view](../images/explore-profiles-homescreen.png)
 
 ### Profile types
 
 The Profile types shows one chart for each profile available for the selected service.
+You can select to display the charts in a grid or one chart per row.
 
 ![Profile types](../images/explore-profiles-profile-types.png)
 
 ### Labels
 
+The Labels view lets you navigate and analyze performance data through tags and labels.
```

Comment:
Aren't tags and labels the same thing? Maybe we can just use "labels", wdyt?

Reply:
Happy to update. Have we removed tags from the UI? I think using tags and labels is a holdover from where we talked about the Tag view, etc.

Reply:
Removed the word tag in most locations in favor of labels.